src/@next/Switch/SwitchStyle.tsx

```diff
@@ -2,6 +2,13 @@ import styled from 'styled-components';
 import { Breakpoints, Spacing, Typography } from '..';
 
 import { Blue, Neutral } from '../utilities/colors';
+import { keyframes } from 'styled-components';
+
+const spin = keyframes`
```

Comment:
can we use the <SpinnerComponent /> ?

Reply:
What do you mean. Can you elaborate in the comment? @Afsal102

Reply:
Cannot. I tried, but the problem is I have to replace the knob with the loading. The knob is declared in the style and If I want to use the <Spinner/> It has to be on the component. 

Even I try to do 
```
export const spin = styled(Spinner)``;
```

I cannot use it, the animation like this because it will not work.

```
  &[data-loading='true']::before {
    border: 2px solid ${Neutral.B100};
    border-top-color: transparent;
    background: transparent;
    animation: ${spin};
    transform: rotate(0deg);
  }
```